### PR TITLE
Internationalization

### LINKED ISSUE #4: Branch for i18n?
I do not know exactly what that means but I found:
- [~~fnando/i18n-js~~](/fnando/i18n-js) ‒ _for Ruby_
- [jamuhl/i18next](/jamuhl/i18next) ‒ [**i18next.com**](http://i18next.com/) ‒ _it seems be in advanced stage of development_
- [~~mozilla/node-webmaker-i18n~~](/mozilla/node-webmaker-i18n) ‒ _for Node.js_
- [recurser/jquery-i18n](/recurser/jquery-i18n) ‒ _operates as other languages programming_
- [musterknabe/translate.js](/musterknabe/translate.js) ‒ _little abstraction_
- [joehms22/interpret.js](/joehms22/interpret.js) ‒ _little documentation_

If you want to maintain a unified and growing project, you may want to internationalize, to save effort in merging of new features.

### PR DESCRIPTION
**Pull request by sabas**
I tried to port the internationalization part done in #2 .
If deemed good could fix #4 and #7

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,82 +13,76 @@
...
-                    <a>Details</a>
+                    <a href='#details' data-i18n="step2.title"></a>
                 </div>
                 <div class='col4 center pad1 step-3'>
-                    <strong>Done</strong>
+                    <strong data-i18n="step3.title"></strong>
                 </div>
             </div>
-
+
```
Comment (alexandre-mbm) [inline index.html:37]: @sabas, erase this spaces and amend or commit again.

File: index.html [hunk 2]
```diff
@@ -13,82 +13,76 @@
...
-                            <input class='col8' id='address' placeholder='501 West Main Street, Alexandria, VA' />
-                            <button class='col4' id='findme'>Find Me!</button>
+                            <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
+                            <button class='col4' id='findme' data-i18n="step1.button"></button>
                         </form>
                     </div>
                     <br />
                     <br />
                     <div class='col12' id='findme-map'></div>
-                    <p id="couldnt-find" style='display: none;'>
```
Comment (alexandre-mbm) [inline index.html:53]: @sabas, I think that this must be translated.

Comment (sabas) [inline index.html:53]: It's moved on line 41

File: index.html [hunk 3]
```diff
@@ -13,82 +13,76 @@
...
 
             <div class='col12 hide' id='collect-data-step'>
-                <p>We found your location. Now tell us about your business.</p>
+                <p data-i18n="[html]step2.desc"></p>
                 <div class='col12'>
                     <div class='clearfix col12'>
-                        <label for='category'>Category</label>
-                        <input id='category' data-placeholder='Pick a category...' class='col10' />
+                        <label for='category' data-i18n="step2.catlabel"></label>
+						<input id='category' data-placeholder='' class='col10'  data-i18n="[data-placeholder]step2.catplaceholder"/>
```
Comment (alexandre-mbm) [inline index.html:61]: @sabas, take care with spaces and indentation.

File: index.html [hunk 4]
```diff
@@ -13,82 +13,76 @@
...
+                        <input id='phone' type='text' placeholder='' data-i18n="[placeholder]step2.phoneplaceholder"/>
                     </div>
-                    <div>
-                        <label for='website'>Website</label>
-                        <input id='website' type='text' placeholder='http://www.example.com/' />
-                        <label for='twitter'>Twitter</label>
-                        <input id='twitter' type='text' placeholder='@example' />
-                        <label for='opening_hours'>Hours</label>
-                        <input id='opening_hours' type='text' placeholder='Monday-Friday 10am to 8pm' />
+                    <div class='col6'>
```
Comment (alexandre-mbm) [inline index.html:69]: @sabas, is `class='col6'` tested here?

File: index.html [hunk 5]
```diff
@@ -13,82 +13,76 @@
...
             <div class='col12 pad2 hide' id='confirm-step'>
-                <p>
-                    <strong>Thanks!</strong> An OpenStreetMap volunteer will add your data to the map soon.
-                </p>
-                <p>
-                    Did you know you can add your own data to the map and skip the wait?
-                    It's very easy to get started <a href="http://openstreetmap.org/edit">editing OSM</a>
-                    in your browser.
+                <p data-i18n="[html]step3.row1"></p>
+                <p><span data-i18n="step3.row2"></span> <span id='linkcoords'>&nbsp;</span></p>
```
Comment (alexandre-mbm) [inline index.html:83]: Three new lines or formats for test.

File: index.html (hunk 4 above)
Comment (sabas) [inline index.html:69]: http://osmitalia.github.io/onosm.org/

**Comment by CloCkWeRX** [comment]
Any chance of merging this and we clean up the whitespace after? It adds value without too much of a reduction in code quality.

**Pull request merged**